cla_backend/apps/core/fixtures/initial_groups.json

```diff
@@ -38,7 +38,16 @@
                 ["change_article", "knowledgebase", "article"],
                 ["change_articlecategorymatrix", "knowledgebase", "articlecategorymatrix"],
                 ["add_articlecategorymatrix", "knowledgebase", "articlecategorymatrix"],
-                ["delete_articlecategorymatrix", "knowledgebase", "articlecategorymatrix"]
+                ["delete_articlecategorymatrix", "knowledgebase", "articlecategorymatrix"],
+                ["change_note", "guidance", "note"],
+                ["add_note", "guidance", "note"],
+                ["delete_note", "guidance", "note"],
+                ["change_notetagrelation", "guidance", "notetagrelation"],
+                ["add_notetagrelation", "guidance", "notetagrelation"],
+                ["delete_notetagrelation", "guidance", "notetagrelation"],
```

Comment:
is `notetagrelation` a generated name? "relation" seems redundant - `notetag` would be shorter and just as good